src/core/utils/masterswitchapi.js

```diff
@@ -0,0 +1,39 @@
+import ApiRequest from '../http/apirequest';
+
+/**
+ * This class provides access to the Answers Status page. This page indicates
+ * if the front-end for a particular experience should be temporarily disabled
+ * due to back-end issues.
+ */
+export default class MasterSwitchApi {
+  constructor (globalStorage) {
+    this._globalStorage = globalStorage;
+  }
+
+  /**
+   * Checks if the front-end for the given experience should be temporarily disabled.
+   * Note that this check errs on the side of enabling the front-end. If there are any
+   * issues with the resultant network call, those failures are caught. In this failure
+   * case, the assumption is that things are enabled.
+   *
+   * @param {string} apiKey The apiKey of the experience.
+   * @param {string} experienceKey The identifier of the experience.
+   * @returns {Promise<boolean>} A Promise containing a boolean indicating if the front-end
+   *                             should be disabled.
+   */
+  isDisabled (apiKey, experienceKey) {
+    const requestConfig = {
+      apiKey,
+      baseUrl: 'https://answersstatus.pagescdn.com/',
```

Comment:
It's ok to have this as a default, but maybe we allow overriding in the constructor so that if we need to change it for a particular customer, we can do it?

Reply:
We could add something to the constructor that would allow us to override the path to the status.json for a particular customer. I think we'd need to add a new attribute(s) to the SDK configuration as well if we wanted to support such an override on-the-fly, without any code changes to the SDK.